sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/samples/Generated/Samples/Sample_MachineLearningRegistryResource.cs

```diff
@@ -206,7 +206,7 @@ public async Task Update_UpdateRegistryWithUserCreatedAccounts()
         public async Task GetBlobReferenceSasRegistryDataReference_GetBlobReferenceSASDataReference()
         {
             // Generated from example definition: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/examples/DataReference/getBlobReferenceSAS.json
-            // this example is just showing the usage of "RegistryDataReferences_GetBlobReferenceSas" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "RegistryDataReferences_GetBlobReferenceSAS" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `SAS`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-10-01/mfe.json#L1926